arbutil/espresso_utils_test.go

```diff
@@ -123,7 +124,7 @@ func TestParsePayloadInvalidCases(t *testing.T) {
 
 	for _, tc := range invalidPayloads {
 		t.Run(tc.description, func(t *testing.T) {
-			_, _, _, _, err := ParseHotShotPayload(tc.payload)
+			_, _, _, _, err := ParseHotShotPayload(tc.payload, nil)
```

Comment:
can you add a test to show that it will work with new transaction format as well as legacy?